packages/engine/src/avatar/systems/AvatarAnimationSystem.ts

```diff
@@ -267,16 +284,26 @@ const execute = () => {
     if (rightFootTargetBlendWeight) {
       _quat2.copy(rightFootTransform.rotation)
       if (isAvatarFlipped) {
-        _quat2.multiply(rightFootRotation)
+        _quat2.multiply(Q_X_90)
+      } else {
+        /** I don't know why flipped models don't need this foot angle applied, nor why it needs to be halved */
```

Comment:
This is likely because our flipped models happen to only ever be VRM, it would seem raw bones on VRM also use identity quaternions in bind pose, though I'm not sure why *that* is.